c2corg_ui/static/js/maincontroller.js

```diff
@@ -92,3 +97,9 @@ app.MainController.prototype.isPath = function(path) {
 app.MainController.prototype.animateHeaderIcon = function(e) {
   app.utils.animateHeaderIcon(e);
 };
+/**
+ * @export
+ */
+app.MainController.prototype.updateMapResize = function() {
+  this.scope_ .$root.$emit('updateMapResize');
```

Comment:
Me neither. Maybe I should create a special directive for those buttons...but it would be another 50 lines of code and a new file instead of 3 lines in the mainCtrl ;p